protocol/0066-VALW-validator_tendermint_weights.md

```diff
@@ -15,38 +15,69 @@ If the network has no stake at all, then all validators would have equal voting
-1. give the first `4` validators `1000` of self-stake each. Give the last validator `0`. 
-1. wait for the delegation to become active in the next epoch
-1. wait for 1000 blocks to pass
-1. check that Tendermint weights for the first `4` validators are `2500` and the last validator `1`. 
-1. just before epoch 0 ends the last validator self-stakes `500`. 
-1. epoch 1 starts 
-1. it's not yet been 1000 blocks from when the last validator self-staked but we see that Tendermint weights for the first `4` validators are `2222` and the last validator `1111`. 
+2. give the first `4` validators `1000` of self-stake each. Give the last validator `0`. 
+3. wait for the delegation to become active in the next epoch
+4. wait for 1000 blocks to pass
+5. check that Tendermint weights for the first `4` validators are `2500` and the last validator `1`. 
+6. just before epoch 0 ends the last validator self-stakes `500`. 
+7. epoch 1 starts 
+8. it's not yet been 1000 blocks from when the last validator self-staked but we see that Tendermint weights for the first `4` validators are `2222` and the last validator `1111`. 
+
 
 ### Sanity check if everyone unstakes and undelegates (<a name="0066-VALW-005" href="#0066-VALW-005">0066-VALW-005</a>)
 1. set up a network with 5 validators
-1. give each of the validators the following number of self-staked tokens: `2000`
-1. wait for the delegation to become active in the next epoch
-1. check that Tendermint weights for the validators are `2000`
-1. now every validator removes (via undelegate now) their stake and there are no delegations from other parties
-1. wait for 1000 blocks to pass
-1. the Tendermint weights for each validator are `10` each as there is no stake in the network. 
\ No newline at end of file
+2. give each of the validators the following number of self-staked tokens: `2000`
+3. wait for the delegation to become active in the next epoch
+4. check that Tendermint weights for the validators are `2000`
+5. now every validator removes (via undelegate now) their stake and there are no delegations from other parties
+6. wait for 1000 blocks to pass
+7. the Tendermint weights for each validator are `10` each as there is no stake in the network. 
+
+### Validator has 1000 stake but 500 delegated , check only attention to 500 (<a name="0066-VALW-006" href="#0066-VALW-006">0066-VALW-006</a>)
+1. set up a network with 5 validators
+2. give each of the validators the following number of self-staked tokens: `1000`
+3. delegate for the first `4` validators `1000` and only delegate `200` to the last node
+4. wait for the delegation to become active in the next epoch
+5. check that Tendermint weights for the first `4` validators are `2222` and the last validator `11119`.
+
+### Validators without self-delegated, check  (<a name="0066-VALW-007" href="#0066-VALW-007">0066-VALW-007</a>)
+1. set up a network with 5 validators
+2. give each of the validators the following number of self-staked tokens: `0`
```

Comment:
What's the network parameter setting self-delegation minimum?

Reply:
If you mean  "reward.staking.delegation.minimumValidatorStake": "0", 
then its default is 0 and remains 0 for the tests.